tests/rbd/rbd_utils.py

```diff
@@ -237,9 +250,48 @@ def remove_image(self, pool_name, image_name):
         """
         self.exec_cmd(cmd=f"rbd rm {pool_name}/{image_name}")
 
+    def move_image_trash(self, pool_name, image_name):
+        """
+        Move images to trash from the specified pool
+        Args:
+            pool_name : name of the pool
+            image_name : mane of the image
+        Returns:
+
+        """
+        self.exec_cmd(cmd=f"rbd trash mv {pool_name}/{image_name}")
+
+    def remove_image_trash(self, pool_name, image_id):
+        """
+        Remove images from trash using image unique ID
+        Args:
+            pool_name : name of the pool
+            image_id : unique id of the images generated after moving image to trash
+        Returns:
+
+        """
+        self.exec_cmd(cmd=f"rbd trash rm {pool_name}/{image_id}")
+
+    def get_image_id(self, pool_name, image_name):
+        """
+        Returns the image_id of image from the poolname
+        Args:
+            pool_name : name of the pool
+        Returns:
+            image_id of the image
+
+        """
+        out = self.exec_cmd(
+            cmd=f"rbd trash list {pool_name} --format json ", output=True
+        )
+        image_info = json.loads(out)
+        for value in image_info:
+            if value["name"] == image_name:
+                image_id = value["id"]
```

Comment:
we can directly return value["id"] right ? we can save one variable name

Reply:
Done